apps/site/assets/js/fullstory.js

```diff
@@ -4,19 +4,19 @@ export default function fullstory() {
   const MODULO_SAMPLE = 50;
   const id = getUniqueId();
 
-  // Only execute the fullstory code if the id is a multiple of the selected modulo
+  // Only execute the Fullstory code if the id is a multiple of the selected modulo
   if (id % MODULO_SAMPLE == 0) {
     fullStoryInit();
   }
 }
 
-// This is the snippet that was generqated by Full Story. No changes were made to
+// This is the snippet that was generated by FullStory. No changes were made to
 // this auto-generated code
 function fullStoryInit() {
-  window["_fs_debug"] = false;
```

Comment:
If we didn't make any changes to the functionality in this file, can we checkout these changes, for better git history?